man/portablectl.xml

```diff
@@ -391,6 +391,15 @@
         <xi:include href="version-info.xml" xpointer="v245"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--clean</option></term>
```

Comment:
hmm, why not do --clean=… just like systemctl?

And then a shortcut `-c` or so which is equivalent to --clean=all,fdstore?

sounds less "lossy" to me, and less surprising, since the option is just the same.

Reply:
ah, on systemctl it's called --what=, not --clean=. hence maybe ignore me

Reply:
a short switch for this sounds pretty useful still though.